src/main/java/edu/ithaca/dragon/blackjack/BlackJack.java

```diff
@@ -4,23 +4,24 @@
 import java.util.Collections;
 
 public class BlackJack {
-    private final int ID;
+    private final long ID;
     private ArrayList<Player> players;
     private ArrayList<Hand> hands;
     private Deck deck;
 
-    public BlackJack(int IDin, Player playerIn){
+    public BlackJack(long IDin, Player playerIn){
         ID = IDin;
         players = new ArrayList<>();
         players.add(playerIn);
         hands = (ArrayList<Hand>)playerIn.getHands().clone();
         deck = new Deck();
     }
-    public BlackJack(int IDin, ArrayList<Player> playerIn)throws IllegalArgumentException{
```

Comment:
We should hold things as List, not ArrayList. Usually prefer the use of an Interface over the use of a specific data structure.

Reply:
Fixed